manifests/apps/jupyter/create-jupyter-notebook-quickstart.yaml

```diff
@@ -6,65 +6,50 @@ metadata:
-      review:
-        instructions: |-
-          #### To verify you have launched JupyterHub:
-          Is a new **JupyterHub** browser tab visible with the **Start a notebook server** page open?
-        failedTaskHelp: This task is not verified yet. Try the task again.
-      summary:
-        success: You have launched JupyterHub.
-        failed: Try the steps again.
-    - title: Configure and start an environment
-      description: |-
-        ### Configuring and starting an environment:
-        1. Select a notebook image from the options provided.
-        2. Select a container size based on your computation needs.
-        3. Click the **Start** button.
-
-        The **Start a notebook server** page will reload and indicate that the system is starting up.
-      review:
-        instructions: |-
-          #### To verify that you have launched the Jupyter notebook:
-          Do you see a message in the page that says **The server is starting up**?
-        failedTaskHelp: This task is not verified yet. Try the task again.
-      summary:
-        success: Your server has started and the JupyterLab interface will load shortly. When the page displays a **Stop** option, proceed to the next step.
-        failed: Try the steps again.
-    - title: Create your first notebook
-      description: |-
-        ### To create a notebook, follow these steps:
-        1. In the **Launcher** tab, under the **Notebook** heading, click the **Python 3** tile.
-        2. When the new Jupyter notebook opens, verify that you see `Python 3` in the upper right navigation bar.
-
-        You have now launched a Jupyter notebook and can begin writing Python.
+        ### Prerequisites
+        - You have logged in to Open Data Hub.
+        - You know the names and values that you want to use for environment variables in your notebook server environment, for example, `AWS_SECRET_ACCESS_KEY`.
+        - If you want to work with a very large data set, work with your administrator to proactively increase the storage capacity of your notebook server.
+        ### Launch a notebook
+        1. On the **Enabled applications** page, locate the **Jupyter** card.
+        2. Click **Launch application**.
+           a) If prompted, select your identity provider.
+           b) Enter your credentials and click **Log in** (or equivalent for your identity provider).
+           c) If you see an **Access permission needed** message, you are not in the default user group or the default administrator group for OpenShift Data Hub. Contact your administrator so that they can add you to the correct group using [Adding users for OpenShift Data Science](https://access.redhat.com/documentation/en-us/red_hat_openshift_data_science/1/html/managing_users_and_user_resources/adding-users-for-openshift-data-science_useradd).
+        3. If you have previously launched Jupyter, skip to Step 4.
+           If you have not launched Jupyter previouslt, start a notebook server.
```

Comment:
small typo, neds to be "previously"

@andrewballantyne @lucferbux other than that, to me now, all changes look ready to be merged"